src/bundle/Resources/views/themes/admin/language/list.html.twig

```diff
@@ -116,13 +114,16 @@
         {% endfor %}
 
         {% embed '@ibexadesign/ui/component/table/table.html.twig' with {
-            headline: 'language.list'|trans|desc('Languages'),
+            headline: 'language.list_number_of_results'|trans({ '%number_of_results%': pager.nbResults })|desc('List (%number_of_results%)'),
             head_cols: [
                 { has_checkbox: true },
                 { content: 'language.name'|trans|desc('Name') },
                 { content: 'language.code'|trans|desc('Code') },
                 { content: 'language.id'|trans|desc('ID') },
-                { content: 'language.enabled'|trans|desc('Enabled') },
+                {
+                    class: 'ibexa-table__header-cell--content-center',
```

Comment:
This class should be added to the table component. 
We are centring header, but not content of the column?